### PR TITLE
Remove scalac options for 2.11

### PR DESCRIPTION
**Pull request by jw3**
This can go away since #427

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -151,10 +151,6 @@ lazy val commonSettings = Seq(
 
 def scalacCrossOptions(scalaVersion: String) =
```
Comment (jw3) [inline build.sbt:152]: Could strip out the entire function for clarity, or leave it for future convenience.

Comment (stevedlawrence) [reply build.sbt:152]: I think it's worth leaving this in and just removing the 2.11 case. I suspect we'll want to add support for 2.13 at some point and it wouldn't surprise me if scalc options changed again.

**Review by stevedlawrence — APPROVED** [review]
+1 :+1: Thanks for the contribution!

Only minor suggestion is to include a bug number in the commit message. I think DAFFODIL-2404 is reasonable since this is essentially an update to that bug.

File: build.sbt (hunk 1 above)
Comment (jw3) [reply build.sbt:152]: I was leaning that way too :+1:

**Comment by jw3** [comment]
> include a bug number in the commit message. I think DAFFODIL-2404

done

**Review by tuxji — APPROVED** [review]
+1 from me too

I suspect -Ywarn-unused:imports will move to scalacOptions if 2.13 supports it as well, but we may need a 2.13 case for options specific to 2.13 anyway.

**Pull request merged**